services/operatorsinfo/operatorsinfo_inmemory.go

```diff
@@ -128,8 +123,9 @@ func (ops *OperatorsInfoServiceInMemory) startServiceInGoroutine(
 	queryC <-chan query,
 	wg *sync.WaitGroup,
 	opts Opts,
-) {
-	go func() {
+) chan<- error {
```

Comment:
I think it would be more readable/and simplify error handling if we replaced the error channel with an error group here

Reply:
Great idea! I added it.